py3status/modules/backlight.py

```diff
@@ -147,7 +152,15 @@ def on_click(self, event):
             self._set_backlight_level(level)
 
     def _set_backlight_level(self, level):
-        self.py3.command_run(self._command_set(level))
+        if self.command_available:
+            self.py3.command_run(self._command_set(level))
+            return
+        for brightness_max_line in open("%s/max_brightness" % self.device, "rb"):
+            brightness_max = int(brightness_max_line)
```

Comment:
I know you copied this code from below, but what kind of weird syntax is this, we loop over lines and overwrite previous values? 🤔 I wonder if we should make it more readable somehow...

Reply:
well... using a for loop to read all lines of a file is a classic. In this case the file is guaranteed to only contain one line (kernel API) so the cod is correct..

Alternative would  be
```python
with open(...) as f: brightness_max = int(f.read()) 
```
that might be consider cleaner but it's a matter of taste at that point

Reply:
In 22 days, when python2 reaches EOL, this can simply become something like `int(Path(...).read_text())` 😄 

But ok, at least I see how the current code works, if we were to refactor here, let's do it separately anyways.